src/MoBi.Core/Services/SimulationConfigurationFactory.cs

```diff
@@ -57,7 +57,8 @@ public SimulationConfiguration CreateFromProjectTemplatesBasedOn(SimulationConfi
 
          configuration.ModuleConfigurations.Each(moduleConfiguration => { simulationConfiguration.AddModuleConfiguration(templateModuleConfigurationFor(moduleConfiguration)); });
 
-         simulationConfiguration.Individual = templateBuildingBlockFor(configuration.Individual);
+         if (configuration.Individual != null)
+            simulationConfiguration.Individual = templateBuildingBlockFor(configuration.Individual);
```

Comment:
Why are you saving the template and not the clone?

Reply:
That's so the linking all works when you create the model.

This method creates the configuration with the template bb's. Then the model, then they are cloned

Reply:
Oh, that's a bug I found along the way BTW. this crashed if you had a simulation without an individual

Reply:
Ok I was wondering why this change was related to commit. This got ne confused